test/e2e/aks.go

```diff
@@ -303,3 +310,48 @@ func GetWorkingAKSKubernetesVersion(ctx context.Context, subscriptionID, locatio
 
 	return maxVersion, nil
 }
+
+// GetLatestStableAKSKubernetesVersion returns the latest stable available Kubernetes version of AKS.
+func GetLatestStableAKSKubernetesVersion(ctx context.Context, subscriptionID, location, version string) (string, error) {
+	settings, err := auth.GetSettingsFromEnvironment()
+	if err != nil {
+		return "", err
+	}
+	authorizer, err := settings.GetAuthorizer()
+	if err != nil {
+		return "", err
+	}
+	containerServiceClient := containerservice.NewContainerServicesClient(subscriptionID)
+	containerServiceClient.Authorizer = authorizer
+	result, err := containerServiceClient.ListOrchestrators(ctx, location, ManagedClustersResourceType)
+	if err != nil {
+		return "", err
```

Comment:
nit: I prefer to wrap errors to have more context while debugging and viewing logs. 
Example: 
```
result, err := containerServiceClient.ListOrchestrators(ctx, location, ManagedClustersResourceType)
if err != nil {
		return "", errors.Wrap(err, "failed to list orchestrators")
	}
```